Buildings/Resources/Python-Sources/makeEplusBuilding.py

```diff
@@ -0,0 +1,212 @@
+import re
```

Comment:
This file should not be in this directory as the directory is used for files that are invoked from Modelica.
A better place would be `Buildings/Resources/src/Examples/ScalableBenchmarks/ZoneScaling/EnergyPlus`, and put all the idf template files that are required to generate the actual idf file. 

Then, also put some README.md file that explains what files such as `Buildings/Resources/Data/Examples/ZoneScaling/raw/RefBldgLargeOfficeNew2004_Chicago_template_adiabatic.idf` are.
It also should not be in `raw` I think, but maybe in `Resources/src` if it is used to generated the actual idf file.

Lastly, add to the travis CI tests a test that invokes `makeEplusBuilding.py`, and returns an error if the generated file is not the same as the one that is distributed with the library. Otherwise, they get out of sync and we won't notice for a while.

What is the process for updating these files as the EnergyPlus version changes? I assume the '*_template_*' files can be run through the EnergyPlus conversion program, and then `makeEplusBuilding.py` be invoked, but this is not clear.

Reply:
I moved this file and the original and template models to `Buildings/Resources/src/Examples/ScalableBenchmarks/ZoneScaling/`. I have also removed the San Francisco models, which included the adiabatic version. These were created and used when we were iterating on the final product but we don't need to keep them anymore.

I've added a `README.md` that provides info about the utility and the files, and also give instruction about updating the Eplus version.